### PR TITLE
[CWS] fix data race in ptracer

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

Avoid having the syscall message modifying by two goroutines. Serialize before being consumed by multiple groutines

### CODE REVIEW

**Review by paulcacheux — APPROVED** [review]

**Review by lebauce — APPROVED** [review]

File: pkg/security/ptracer/cws.go [hunk 1]
```diff
@@ -218,14 +215,30 @@ func StartCWSPtracer(args []string, envs []string, probeAddr string, opts Opts)
 	}
 
 	var (
-		msgChan        = make(chan *ebpfless.Message, 100000)
+		msgDataChan    = make(chan []byte, 100000)
```
Comment (spikat) [inline pkg/security/ptracer/cws.go:218]: Switching from a number of msg to a number of bytes, IMO we may want to increase the size of the chan

Comment (paulcacheux) [reply pkg/security/ptracer/cws.go:218]: I don;t think that's the case. The chan is a chan of []byte, so the size applies to the count of slices, not underlysing bytes

Comment (spikat) [reply pkg/security/ptracer/cws.go:218]: missread

**Review by spikat — APPROVED** [review]

**Pull request merged**